pkg/controller/issueapi/codestatus.go

```diff
@@ -68,13 +68,15 @@ func (c *Controller) HandleCheckCodeStatus() http.Handler {
 			return
 		}
 
+		logger.Debugf("Found code: %#v", code)
```

Comment:
Debug intended?

Reply:
it was, but we could go with info?

Reply:
Whatever we do, lets use structured logs with Debugw or Infow